tensorflow/docs_src/programmers_guide/graphs.md

```diff
@@ -505,10 +505,10 @@ multiple graphs in the same process.
 As noted above, TensorFlow provides a "default graph" that is implicitly passed
 to all API functions in the same context. For many applications, a single graph
 is sufficient. However, TensorFlow also provides methods for manipulating
-the default graph, which can be useful in more advanced used cases. For example:
+the default graph, which can be useful in more advanced use cases. For example:
 
 * A @{tf.Graph} defines the namespace for @{tf.Operation} objects: each
-  operation in a single graph must have a unique name. TensorFlow will
+  operation in a single graph must have an unique name. TensorFlow will
```

Comment:
`a unique` is correct. The rule is to use `an` for words where the first syllable is a vowel sound. The first syllable of `unique` has a consonant sound, so `a` is correct.